webpack.config.js

```diff
@@ -67,6 +67,7 @@ module.exports = {
     },
     resolve: {
         extensions: ['.js', '.jsx'],
+        symlinks: false,
```

Comment:
Just curious, why?

Reply:
This enables using `npm link` to work with a local version of `pc-nrfconnect-shared`. I will describe how to do this in https://nordicsemiconductor.github.io/pc-nrfconnect-docs/core_development when the major work of the redesign is done.